.github/workflows/python-test.yml

```diff
@@ -36,9 +36,9 @@ jobs:
           python -m pip install tox tox-gh-actions
       - name: Test with tox
         run: tox
-      - name: Prepare artifacts
-        run: mkdir -p .coverage-data && mv .coverage.* .coverage-data/
-      - uses: actions/upload-artifact@master
-        with:
-          name: coverage-data
-          path: .coverage-data/
+      # - name: Prepare artifacts
```

Comment:
This no longer work in CI

Reply:
**Question (non-blocking)** What were the failures? If this is just for Oxfam then I'm not too worried by the change; but it'd be good to get the other changes into our main branch if we can sort that out at some point.